README.md

```diff
@@ -72,7 +72,7 @@ Copy WowPacketParser/App.config as template and edit as your needs.
 To run:
 
 ```
-docker run -v /place/where/sniffs/are/kept:/sniffs -v App.config:/app/WowPacketparser.dll.config trinitycore/wpp /sniffs/sniffname.pkt
+docker run --rm -it -v /place/where/sniffs/are/kept:/sniffs -v /place/where/config/is/App.config:/app/WowPacketParser.dll.config trinitycore/wpp /sniffs/sniffname.pkt
```

Comment:
why are we removing where sniffs are kepts? Or am I miss understanding that arg string?

Reply:
Remove that change from this PR, it has nothing to do with adding new build

Reply:
This change leaked. I will fix.